src/BuildScriptGenerator/Node/NodeManifestFilePropertyKeys.cs

```diff
@@ -0,0 +1,16 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Microsoft.Oryx.BuildScriptGenerator.Node
+{
+    public class NodeManifestFilePropertyKeys
+    {
+        /// <summary>
+        /// The name of the property whose value indicates the location of the output directory of a Node app.
+        /// For example, 'dist' or 'public' folder.
+        /// </summary>
+        public const string OutputDirPath = nameof(OutputDirPath);
```

Comment:
is it really the output, or just some specific component like static content?

Reply:
As I mentioned in scrum today, this has been the typical naming pattern by different frameworks, so I kept it this way.